controllers/admin/AdminLinkWidgetController.php

```diff
@@ -218,11 +219,45 @@ protected function buildHelper()
         return $helper;
     }
 
-    public function validateForm($data)
+    /**
+     * @param array $fields
+     * @return bool
+     */
+    public function validateLinkName(array $fields)
```

Comment:
to be more specific, please take a look here:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/AdminController.php#L1094

and here:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/AdminController.php#L3441